### PR TITLE
MAT-6224 generate HQMF, add it to package 

### PR DESCRIPTION
**Pull request by adongare**
## MADiE PR

Jira Ticket: [MAT-6224](https://jira.cms.gov/browse/MAT-6224)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/services/HqmfService.java [hunk 1]
```diff
@@ -16,12 +18,27 @@ public class HqmfService {
...
+    } catch (JAXBException ex) {
+      String message =
+          "An issue occurred while generating the simple xml for measure: " + qdmMeasure.getId();
+      log.error(message, ex);
+      throw new PackagingException(message, ex);
+    }
 
-    return hqmfGenerator.generate(measureExport);
+    Generator hqmfGenerator = hqmfGeneratorFactory.getHQMFGenerator();
+    try {
```
Comment (RohitKandimalla) [inline src/main/java/gov/cms/madie/services/HqmfService.java]: Do you think we can use a single Try block but with multiple catch blocks ? Looks like the Exceptions we are catching are different, so I believe we can have multiple catch blocks

Comment (adongare) [reply src/main/java/gov/cms/madie/services/HqmfService.java]: we could, updated.

File: src/main/java/gov/cms/madie/services/MeasureMapper.java [hunk 2]
```diff
@@ -211,7 +211,7 @@ default List<Object> populationToDefOrAgg(Population population) {
 
   @Named("scoringUnitToUcum")
   default String scoringUnitToUcum(Object scoringUnit) {
-    if (scoringUnit == null) {
+    if (scoringUnit == null || "".equals(scoringUnit)) {
```
Comment (nmorasb) [inline src/main/java/gov/cms/madie/services/MeasureMapper.java:214]: let's swap this for StringUtils.isBlank

Comment (nmorasb) [reply src/main/java/gov/cms/madie/services/MeasureMapper.java:214]: which we can do with the next PR since we're still working in this..

**Review by nmorasb — APPROVED** [review]

File: src/main/java/gov/cms/madie/services/MeasureMapper.java (hunk 2 above)
Comment (adongare) [reply src/main/java/gov/cms/madie/services/MeasureMapper.java:214]: It's not string always. if value present, it is an object. if not, it could be null or empty string. this needs to be handled in UI while adding/editing to behave consistency.
